gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -171,16 +171,14 @@ gpii.contextManager.contextChanged = function (that, lifecycleManager, forceCont
     var fullPayload = fluid.extend(true, {}, activeSession.model);
     var oldActiveContext = fullPayload.activeContextName;
 
-    fluid.log("contextManager starts to apply a context change from the old context \"", oldActiveContext, "\" to the new context \"", forceContextName, "\" for the GPII key - ", activeGpiiKeys[0]);
-
     // find and update context:
     that.updateActiveContextName(fullPayload, forceContextName);
 
     if (oldActiveContext === fullPayload.activeContextName) {
         fluid.log(fluid.logLevel.TRACE, "contextManager: Same context as before (" + oldActiveContext + ") so doing nothing");
         return;
     }
-    fluid.log("contextManager: New active contexts: " + fullPayload.activeContextName);
+    fluid.log("contextManager: Apply a context change from the old context \"", oldActiveContext, "\" to a new active context \"" + fullPayload.activeContextName, "\" for the GPII key: ", activeGpiiKeys[0]);
```

Comment:
The present tense wording of these messages does sound a little odd.
How about, "Context changed from old context " .... "  to new active context"  ....